algorithms_and_computer_science/letters_and_words/questions_and_answers.js

```diff
@@ -1,5 +1,5 @@
 function getAnswer(entry) {
-  // Put your code here.
+  return entry.split(": ")[1];
```

Comment:
Oops!  Make sure you run `git status` and `git diff` so that you don't accidentally commit changes you didn't intend.

In this case, we can't merge this into the master repo.  You should close this PR and re-open with the correct commit.